aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/build/BuildTriggerStepTest.java

```diff
@@ -165,4 +170,25 @@ public void cancelBuildQueue() throws Exception {
         j.assertLogContains("branch=release extra=true", ds.getBuildByNumber(3));
     }
 
+    @Issue("JENKINS-26123")
+    @Test public void noWait() throws Exception {
+        j.createFreeStyleProject("ds").setAssignedLabel(Label.get("nonexistent"));
+        WorkflowJob us = j.jenkins.createProject(WorkflowJob.class, "us");
+        us.setDefinition(new CpsFlowDefinition("build job: 'ds', wait: false"));
+        j.assertBuildStatusSuccess(us.scheduleBuild2(0));
```

Comment:
@jtnord I think it's the downstream job that fails, which should be irrelevant to the Workflow job since it's not waiting on (`wait: false`), right?